packages/databricks-vscode/src/test/e2e/utils.ts

```diff
@@ -204,3 +205,38 @@ export async function startSyncIfStopped() {
         }
     );
 }
+
+export async function getTabByTitle(title: string) {
+    const workbench = await browser.getWorkbench();
+    return await browser.waitUntil(
+        async () => {
+            console.log("Searching for a tab with title:", title);
+            const tabs = await workbench.getEditorView().getOpenTabs();
+            for (const tab of tabs) {
+                const tabTitle = await tab.getTitle();
+                console.log("Found a tab:", tabTitle);
+                if (tabTitle.includes(title)) {
+                    return tab;
+                }
+            }
+        },
+        {timeout: 5000}
+    );
+}
+
+export async function waitForInput() {
+    const workbench = await browser.getWorkbench();
+    let input: InputBox | undefined;
+    await browser.waitUntil(
+        async () => {
+            input = await new InputBox(workbench.locatorMap).wait();
+            return input !== undefined;
+        },
+        {timeout: 5000, interval: 500}
+    );
+    assert(input !== undefined);
+    while (await input.hasProgress()) {
+        await sleep(500);
+    }
```

Comment:
IMO also use timeout? Easier to debug it from logs. Otherwise the calling function might timeout and then we have to repro to find the line that timedout. 

Not sure what the best practice is here though. What is the best granularity to have timeouts in tests?